_posts/2021-06-03-opensearch-release-candidate-announcement.markdown

```diff
@@ -0,0 +1,79 @@
+---
+layout: post
+title:  "OpenSearch Release Candidate (RC1) is now available"
+authors: 
+  - andhopp
+date:   2021-06-07
+categories: update
+---
+
+Today we are excited to announce the Release Candidate (RC1) for version 1.0.0 of both OpenSearch (derived from Elasticsearch 7.10.2) and OpenSearch Dashboards (derived from Kibana 7.10.2). The Release Candidate includes downloadable [artifacts](https://opensearch.org/downloads.html) (Linux tars and Docker images) as well as a number of OpenSearch plugins (listed below). 
```

Comment:
This is a single author blog post. Using collective 'we' is something we should get away from in blog posts.

Reply:
Done, thanks!